site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -0,0 +1,202 @@
+---
+api: sidePanel
+---
+
+## Overview {: #overview }
+
+Chrome features a built-in side panel that enables users to view more information alongside the main content of a webpage. The SidePanel API allows extensions to add their own panels to the Chrome browser. Some features include:
+
+- The side panel remains open when navigating between tabs (if set to do so).
+- It can be available only on specific websites.
+- As an extension page, side panels have access to all Chrome APIs.
+- Users can choose which side to display the panel under Chrome settings.
+
+## Manifest {: #manifest }
+
+To use the SidePanel API, add the `"sidePanel"` permission in the extension [manifest][doc-manifest] file:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "name": "My side panel extension",
+  ...
+  "permissions": [
+    "sidePanel"
+  ]
+}
+```
+
+## Use cases {: #use-cases }
+
+The following sections demonstrate some common use cases for the SidePanel API. See [Extension samples](#examples) for complete extension examples.
+
+### Display the same side panel on every site {: #every-site }
+
+The side panel can be set initially from the `“default_path”` property in the `“side_panel”` key of the manifest to display the same side panel on every site. This should point to a relative path within the extension directory.
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "name": "My side panel extension",
+  ...
+  "side_panel": {
+    "default_path": "sidepanel.html"
+  }
+  ...
+}
+```
+
+{% Label %}sidepanel.html:{% endLabel %}
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Sidepanel</title>
+  </head>
+  <body>
+    <h1>All sites sidepanel extension</h1>
+    <p>This side panel is enabled on all sites</p>
+  </body>
+</html>
+```
+
+### Enable a side panel on a specific site {: #by-site }
+
+An extension can use [`sidepanel.setOptions()`][sidepanel-setoptions] to enable a side panel on a specific site. This example uses [`chrome.tabs.onUpdated()`][tabs-onupdated] to listen for any updates made to the tab. It checks if the URL is [www.google.com](https://www.google.com) and enables the side panel. Otherwise, it disables it. 
+
+{% Label %}service-worker.js:{% endLabel %}
+
+```js
+const googleURL = 'https://www.google.com';
+
+chrome.tabs.onUpdated.addListener((tabId, info, tab) => {
+  if (!tab.url) return;
+  const url = new URL(tab.url);
+  // Enables the side panel on google.com
+  if (url.origin === googleURL) {
+    chrome.sidePanel.setOptions({
+      tabId,
+      path: 'sidepanel.html',
+      enabled: true
+    });
+  // Disables the side panel on all other sites
+  } else {
+    chrome.sidePanel.setOptions({
+      tabId,
+      enabled: false
+    });
+  }
+});
+```
+
+When the user navigates to a site where the side panel is not enabled, the side panel will close and not be available on the side panel drop-down menu.
```

Comment:
```suggestion
When a user temporarily switches to a tab where the side panel is not enabled, the side panel will be hidden. It will automatically show again when the user switches to a tab where it was previously open.

When the user navigates to a site where the side panel is not enabled, the side panel will close and the extension will not show in the side panel drop-down menu.
```